media-gfx/qimgv/qimgv-0.8.5.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+DESCRIPTION="A cross-platform image viewer with webm support, written in qt5"
+HOMEPAGE="https://github.com/easymodo/qimgv"
+SRC_URI="https://github.com/easymodo/qimgv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="exif kde video"
+
+BDEPEND="
+	dev-util/cmake
```

Comment:
`cmake-utils.eclass` already puts cmake into BDEPEND, so you dont need to specify it here. But ebuild looks good, so remember it next time.